Sources/SwiftInspectorVisitors/Tests/NestableTypeVisitorSpec.swift

```diff
@@ -633,6 +634,15 @@ final class NestableTypeVisitorSpec: QuickSpec {
             expect(matching.count) == 1
           }
 
+          it("finds FooClass.BarFooStruct.BarBarFooTypealias") {
+            let matching = self.sut.typealiases.filter {
+              $0.name == "BarBarFooTypealias"
+                && $0.initializer?.asSource == "Void"
```

Comment:
What is the "initializer" of a typealias?

Reply:
an `initializer` of a typealias is effectively the code between the `=` and the `where` clause of a typealias. If there is no `where` clause, then it's the code after `=`.